Remove repeated content

amal-ghamdi opened the pull request:
closes #47

jakobsj reviewed (changes requested):
Really nice reorganization and simplification. The book is really starting to look excellent!

Some minor suggestions (not sure if I'm viewing the completely latest version?):

- [x] There is still a "remarks" section that reads as if belong to the June course here https://amal-ghamdi.github.io/CUQI-Book-test/chapter01/introduction_overview_of_cuqipy.html
- [x] Put our names as authors on the front page
- [x] What is called chapter 3 in the menu has a filename called chapter A, probably good to rename as chapter 3 for simplicity: https://amal-ghamdi.github.io/CUQI-Book-test/chapter0A/chapter0A.html
- [ ] Chapter 1 startpage has chapter learning objectives (https://amal-ghamdi.github.io/CUQI-Book-test/chapter01/chapter01.html), suggest to add a sentence or two before the learning objective explaining what the chapter is about. 
- [ ] Then other chapters have empty startpages eg chapter 2 would benefit from also having a short intro and chapter learning objectives added: https://amal-ghamdi.github.io/CUQI-Book-test/chapter02/chapter02.html
- [ ] The five little stories are really great as it nicely conveys insights about different samplers. I wonder if it would be beneficial to organize as five separate, shorter files, to keep more manageable and allow navigating them in menu?

amal-ghamdi commented:
Many thanks @jakobsj for your review. I addressed your comments. I implemented the first 3 and copied the last three in another issue #57 (#56). 

I also added some instructions on how to run the book interactively, but this will be possibly revisited as part of issue #53

The test book reflects the changes in this PR:  https://amal-ghamdi.github.io/CUQI-Book-test/

- [ ] One last note, I am not sure about author's order, so we can discuss that

jakobsj commented:
@amal-ghamdi is there a rendered version I could look at?

amal-ghamdi commented:
> https://amal-ghamdi.github.io/CUQI-Book-test/

@jakobsj yes I pushed the same branch to the test book and merged it to the test book main https://amal-ghamdi.github.io/CUQI-Book-test/ (this should be identical to what will this PR produce)

jakobsj reviewed (approved):
Thanks for addressing the comments. Happy with the changes and good to see the remaining ones transferred.

chaozg commented on chapter01/introduction_overview_of_cuqipy.md:
```diff
@@ -101,29 +101,13 @@ See setting up a Bayesian model in CUQIpy in 4 steps [here](https://cuqi-dtu.git
-![toolbox](../images/toolbox3.jpeg)
-
 
 ### How to run the code and solve the coding exercises in this mini-book? <a class="anchor" id="running-the-code"></a>
```
Just a thought: this is a vital piece of information for using this book, and it's a pit that it is somehow hidden in a section called "Overview of CUQIpy". Do you think it deserves a more noticeable place, for example somewhere before chapter01?

amal-ghamdi replied:
Thank you @chaozg, this is a good point, I made a note of it in #68

chaozg commented on chapter07/gradient_based_sampling_with_cuqipy.ipynb:
The file name `gradient_based_sampling_with_cuqipy.ipynb` is a bit misleading (to ourselves when editing) as this notebook contains more "stories" than just gradient sampling. What do you think?

chaozg reviewed (approved):
I think this PR is very nice and it does more than the original purpose, "removing repeated content". Well done @amal-ghamdi .
I'll giving approval now and only leave some tiny thoughts in the comments.

amal-ghamdi commented:
thank you @jakobsj and @chaozg for your review! I am merging. 

@chaozg, I took the liberty to resolve the two conversations from your comments since I tracked it elsewhere (in #68 ).
